app.rb

```diff
@@ -1,8 +1,46 @@
 require 'sinatra/base'
+require_relative 'lib/player.rb'
+require_relative 'lib/game.rb'
 class RockPaperScissors < Sinatra::Base
+  enable :sessions
   get '/test' do
     'test page'
   end
 
+  get '/' do
+    erb :index
+  end
+
+  get '/play' do
+    @player_1_name = $player_1.name
+    erb :play
+  end
+
+  get '/rock' do 
```

Comment:
Most of the code is shared between those 3 endpoints, as they do the same thing - the only difference is the move chosen by the player. You could factor all three into the same endpoint (something like `get '/move`') and use a request parameter to indicate which move/weapon to use. If you do that, you might also want to double check the move is valid, as it comes from "outside" the program (for example the value "cookie" wouldn't make sense and should be ignored/rejected)